primitives/evm/src/lib.rs

```diff
@@ -56,12 +56,113 @@ pub struct Vicinity {
 	pub origin: H160,
 }
 
+/// Maximum allowed POV size for a parachain block.
+pub const MAX_POV_SIZE: u64 = 5 * 1024 * 1024;
```

Comment:
This should not be a frontier constant, should be configurable by the final project